crates/api/src/lib.rs

```diff
@@ -22,18 +24,26 @@ pub trait Perform {
 }
 
 /// Converts the captcha to a base64 encoded wav audio file
-pub(crate) fn captcha_as_wav_base64(captcha: &Captcha) -> String {
+pub(crate) fn captcha_as_wav_base64(captcha: &Captcha) -> Result<String, Box<dyn std::error::Error>> {
```

Comment:
Look at the error type for the other functions in this file.

Reply:
OK, switched to LemmyError and added a translation: https://github.com/LemmyNet/lemmy-translations/pull/76